test/integration/integration_test_fixtures.py

```diff
@@ -153,6 +153,8 @@ def tearDown(self, caplog):
       for record in caplog.get_records(when):
         if record.levelno not in (logging.WARNING, logging.ERROR):
           continue
+        if "Unable to use runtime singleton for feature" in record.message:
```

Comment:
We already have a way of skipping some warnings:

https://github.com/envoyproxy/nighthawk/blob/e2f5265b3e4f0abf7d749c3a0988fb57fe50cea6/test/integration/nighthawk_test_server.py#L63-L69

Can we utilize this one?

Reply:
Done.